custom_exceptions.py

```diff
@@ -64,3 +64,6 @@ class ValidateEnvVariablesError(Exception):
 
 class ServerError(Exception):
     pass
+
+class OpenApiError(Exception):
```

Comment:
This could be renamed to match the responsibility of the `microscope_info.py` scrit. Ex: `MircoscopeQueryError`.

Reply:
Done